spec/es_spec_helper.rb

```diff
@@ -1,15 +1,20 @@
 require_relative './spec_helper'
 
 require 'elasticsearch'
-require_relative "support/elasticsearch/api/actions/delete_ilm_policy"
-require_relative "support/elasticsearch/api/actions/get_alias"
-require_relative "support/elasticsearch/api/actions/put_alias"
-require_relative "support/elasticsearch/api/actions/get_ilm_policy"
-require_relative "support/elasticsearch/api/actions/put_ilm_policy"
 
 require 'json'
 require 'cabin'
 
+# remove this condition and support package once plugin starts consuming elasticsearch-ruby v8 client
+# in elasticsearch-ruby v7, ILM APIs were in a separate xpack gem, now directly available
+unless elastic_ruby_v8_client_available?
+  require_relative "support/elasticsearch/api/actions/delete_ilm_policy"
+  require_relative "support/elasticsearch/api/actions/get_alias"
+  require_relative "support/elasticsearch/api/actions/put_alias"
+  require_relative "support/elasticsearch/api/actions/get_ilm_policy"
+  require_relative "support/elasticsearch/api/actions/put_ilm_policy"
```

Comment:
With this refactoring we no longer need the files related to aliases, which can be deleted from the branch and the require removed too:

```suggestion
  require_relative "support/elasticsearch/api/actions/delete_ilm_policy"
  require_relative "support/elasticsearch/api/actions/get_ilm_policy"
  require_relative "support/elasticsearch/api/actions/put_ilm_policy"
```

Reply:
Ah nice catch, thanks. Removed them with their usages.